docs/source/support.html.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Support
+layout: documentation
+---
+
+## Need help?
+
+Algolia Places is built by [Algolia's people](https://github.com/orgs/algolia/people). We are here to help you with any issues you would face while using Algolia Places.
+
+### Irrelevant results
+
+If it happen that the places suggestions you get seems irrelevant, <a href="https://docs.google.com/forms/d/13r_7B72v6u6326atqzKZpv0fs_3OUOJFR-6QDipHl3Y/viewform?entry.1560244398&entry.1894094686&entry.1809496416&entry.2029760924&entry.1442423869&entry.1714224469&entry.1070945708=Find+it+on+https://api.ipify.org/&entry.2019626860=Find+it+on+https://jsfiddle.net/qmjet97b/" id="support-google-form" target="_blank">let us know via our google form</a>.
```

Comment:
`s/seems/seem`? Also, maybe better to reword `Should the Places suggestions appear irrelevant, ...`